app/controllers/ops_controller/settings/common.rb

```diff
@@ -216,7 +212,7 @@ def pglogical_save_subscriptions
         MiqRegion.replication_type = replication_type
       rescue StandardError => bang
         add_flash(_("Error during replication configuration save: ") %
-                    {:message => bang.message}, :error)
+                      {:message => bang.message}, :error)
```

Comment:
it looks like copy/paste from different environments introduced this auto formatting, will change back